plugins/orchestrator/src/components/WorkflowsTable.tsx

```diff
@@ -130,12 +139,14 @@ export const WorkflowsTable = ({ items }: WorkflowsTableProps) => {
   );
 
   return (
-    <OverrideBackstageTable<FormattedWorkflowOverview>
-      title="Workflows"
-      options={options}
-      columns={columns}
-      data={data}
-      actions={actions}
-    />
+    permittedToReadWorkflows && (
```

Comment:
I think we need some kind of message if the user is not permitted to read the workflows otherwise it won't be clear what's going on

Reply:
I think we can just leave this as is
If the user can't access he'll see an error message on top, it's the same effect only the user will know what's going on

Reply:
Yeah, agree, the big "403" error in red should be enough

Reply:
When I said leave it as is I meant without the change in this PR
With the change in this PR there won't be any error message and nothing will be seen

Reply:
ah, I see what you mean, I'll remote the condition